src/wp-includes/class-wp-theme-json.php

```diff
@@ -2593,8 +2637,9 @@ protected static function remove_insecure_settings( $input ) {
 		$output = array();
 		foreach ( static::PRESETS_METADATA as $preset_metadata ) {
 			foreach ( static::VALID_ORIGINS as $origin ) {
-				$path_with_origin = array_merge( $preset_metadata['path'], array( $origin ) );
-				$presets          = _wp_array_get( $input, $path_with_origin, null );
+				$path_with_origin   = $preset_metadata['path'];
```

Comment:
Can we use `$path_with_origin = array( value1, value2 )` here as well?

Reply:
Similar to https://github.com/WordPress/wordpress-develop/pull/3555#discussion_r1019751990, that would change the data, since `$preset_metadata['path']` is an array itself, so you would end up with an array in an array.